pallets/stateful-storage/src/benchmarking.rs

```diff
@@ -11,7 +11,7 @@ use frame_system::RawOrigin;
 use sp_core::bounded::BoundedVec;
 use stateful_child_tree::StatefulChildTree;
 
-pub const ITEMIZED_SCHEMA: SchemaId = 100; // keep in sync with mock.rs. TODO: refactor
+pub const ITEMIZED_APPEND_ONLY_SCHEMA: SchemaId = 100; // keep in sync with mock.rs. TODO: refactor
```

Comment:
these are refactored. Please make sure to use the `test_common.rs` in the pallet.